rust/rope/src/tree.rs

```diff
@@ -196,6 +196,19 @@ impl<N: NodeInfo> Node<N> {
         }
     }
 
+    fn next_positive_measure_child<M: Metric<N>>(&self, j: usize) -> (Option<usize>, usize) {
```

Comment:
Would prefer a doc comment. Doesn't need to be detailed, as it's fairly clear, but should probably explain the return type.